tests/result/ssdp-m-search-ua.pcap.out

```diff
@@ -0,0 +1,8 @@
+Guessed flow protos:	0
+
+DPI Packets (UDP):	1	(1.00 pkts/flow)
+Confidence DPI              : 1 (flows)
+
+SSDP	4	864	1
+
+	1	UDP 192.168.242.50:56446 -> 239.255.255.250:1900 [proto: 12/SSDP][ClearText][Confidence: DPI][cat: System/18][4 pkts/864 bytes -> 0 pkts/0 bytes][Goodput ratio: 80/0][3.00 sec][PLAIN TEXT (SEARCH )][Plen Bins: 0,0,0,0,0,100,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0,0]
```

Comment:
UserAgent is not logged.. you probably need to update `example/reader_util.c:process_ndpi_collected_info()` and add a new "if" branch for SSDP where you have to save the useragent...

Reply:
I looked into it and it's more work than I have time for at the moment...  I don't use the example code (ndpiReader, etc).  So if it's a requirement to update that code too, feel free to reject this PR.  I maintain my own fork, so it's not a problem for me either way.

Reply:
Ok, I'll do the missing work. Thanks for the contribution

Reply:
Thanks and sorry for the hassle.  I know you're busy too.